### ISSUE TITLE
Screenshot Feature In 3D Viewer

### ISSUE THREAD

**Issue by TayMcKenzieNZ** [DurtyFree/durty-cloth-tool#332]
Would it be possible to have Durty Cloth Tool's 3D Model Viewer take screenshots, and when you set the tool up, you give it an output location for these screenshots? Or would you have to work out some permission thing with Windows?

Personally I just use Windows Snipping Tool in Windows 11 and select the area I want to snip or record a video, but if there was a built in button and keyboard shortcut, that could be a neat idea.

Perhaps F11 could full screen the previewer and Print Screen could capture the screen and a little notification would pop up in your toolbar / notifications, or a window allowing you to go to that file location to view it.

You could also select whether you want jpg or PNG.

**Comment by TayMcKenzieNZ** [comment]
You may also need to have the 3D viewer on your main display for it to work however I'm not entirely sure

**Comment by KilluaZoldyck0099** [comment]
Yeah, why not? If it's to create transparent images for every texture and model and save them, then it'd be a nice tool! I'd use it and a lot of others would to

**Comment by KilluaZoldyck0099** [comment]
> Would it be possible to have Durty Cloth Tool's 3D Model Viewer take screenshots, and when you set the tool up, you give it an output location for these screenshots? Or would you have to work out some permission thing with Windows?
> 
> Personally I just use Windows Snipping Tool in Windows 11 and select the area I want to snip or record a video, but if there was a built in button and keyboard shortcut, that could be a neat idea.
> 
> Perhaps F11 could full screen the previewer and Print Screen could capture the screen and a little notification would pop up in your toolbar / notifications, or a window allowing you to go to that file location to view it.
> 
> You could also select whether you want jpg or PNG.

Yeah you'd give a location output. what if someone has so many clothes that this is so time-consuming that it needs about 3 months to do this? I've done this I spend 3 months doing this so I know this is really time consuming but with something like this this would be easy

**Comment by DurtyFree** [comment]
This is now available in the V4 Experimental Branch, grab it here if you want to try it out: https://gta.clothing/v4
